api/models/transaction.js

```diff
@@ -2,25 +2,33 @@
 var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
-
 var TransactionSchema = new Schema({
 
+    betId: {
+        type: String,
+        required: 'betId is required'
+    },
+
+
     playerId: {
         type: String,
         required: 'playerId is required'
     },
 
     amount: {
-        type: Number,
+        type: Schema.Types.Decimal128,
         required: 'amount is required'
     },
 
-    status: {
-        type: [{
-            type: String,
-            enum: ['initialized', 'cancelled', 'voided', 'completed', 'failed']
-        }],
-        required: 'type is required'
+    hasWon: {
```

Comment:
Czy "transakcja" może wygrać? Wydaje mi się, że mieszamy tutaj bety z transakcjami portfela. Może po prostu nazwiemy to betem, będziemy tutaj trzymać status płatności, a pojedyncze transakcje pieniężne będziemy trzymać tylko w hstorii portfela?

Reply:
Teraz jest Transaction i WalletTransaction. Zgadzam się, że pierwszy można zmienić na BetPayment albo coś podobnego.